src/Student.java

```diff
@@ -9,6 +9,9 @@
  * @todo Implement everything
  */
 public class Student{
+	public static final float WEIGHT_PROPORTIONALITY_CONSTANT = 1;
+	public static final float WEIGHT_THRESHOLD = 10;
```

Comment:
Point 3 of weight algorithm adjustment

This would act more as a limit/ceiling and isn't the best way to adjust it.
> Think of it as clipping when you try to amplify a sound wave